.github/workflows/pr-ci.yml

```diff
@@ -81,7 +81,11 @@ jobs:
       - name: Install Xe
```

Comment:
I would change this to "Ze (oneAPI Level Zero).

Reply:
Updated